### PR TITLE
Move executable scripts to bin/ (part 1)

### PR DESCRIPTION
**Pull request by MrPetovan**
Part of #4629 

By the way, don't hesitate to point out the scripts that don't serve any further purpose.

Addon counterpart: https://github.com/friendica/friendica-addons/pull/551

### CODE REVIEW

**Comment by annando** [comment]
Is ```bin``` the right word? These aren't binaries but scripts.

**Comment by MrPetovan** [comment]
In PHP projects where you will rarely encounter an actual binary file except PHAR files, yes. If you want too nitpick, the `scripts` folder was not appropriate either since the entire project is made of scripts. We might as well go with the popular choice.

**Comment by annando** [comment]
Is there some general rule? Nextcloud for example does have a single file named ```console.php``` in the root for that. Same does Movim. But I prefer having this separated and not mixed up like in these projects.

Best would be - of course - if there had been one general definition (PSR).

**Comment by MrPetovan** [comment]
There is no written rule, but the `bin` folder is common amongst popular PHP projects, while we are the only one sporting a `scripts` and a `util` folder both containing executable scripts.

File: CHANGELOG [hunk 1]
```diff
@@ -436,7 +436,7 @@ Version 3.4.1 (2015-07-06)
 	Fix spelling in accepted connection notification email (strk)
 	Show image size warning in a human readable format (rabuzarus)
 	Move ACL window in template (rabuzarus)
-	New option "-s" in util/run_xgettext.sh (fabrixxm)
```
Comment (tobiasd) [inline CHANGELOG]: I don't think this should be changed in the historic documentation of the CHANGELOG as there will be a new entry with the correct release the the binaries were moved.

**Comment by tobiasd** [comment]
I don't really like the mixture of the stuff needed by the core to function and the utility scripts that nobody needs, but some crazy people to develop the server.

File: CHANGELOG (hunk 1 above)
Comment (MrPetovan) [reply CHANGELOG]: Reverted.

**Comment by MrPetovan** [comment]
I am this close to create a `friendica-dev` package. Don't tempt me!

**Pull request merged**